### DISCUSSION TITLE
How to use capslock as super key(like 2nd hyper key)?

### DISCUSSION THREAD

**Discussion by 1m-N00b** [hyprwm/Hyprland#2494] [Q&A]
i swap caps and ctrl_left with via/qmk.
In other words, since hyper(win) and super(caps) are next to each other, we want to use these two keys.

**Comment by donovanglover** [comment]
You can achieve this with [dual-function-keys](https://gitlab.com/interception/linux/plugins/dual-function-keys). I currently have caps lock mapped to both escape and left ctrl, which you can see [here](https://github.com/donovanglover/nix-config/blob/76bfdf10b65f25615c626c6af068394a60830038/modules/dual-function-keys.nix#L20-L22).

In order to use caps lock as a hyper key, you can read the discussion on GitLab here: https://gitlab.com/interception/linux/plugins/dual-function-keys/-/issues/12

**Comment by PixeKnight** [comment]
ctrl:hyper_capscontrol

**Reply by PixeKnight** [reply]
Not quite what you wanted and I know that this is an old thread. It turns capslock into ctrl, and control into the hyper key

**Reply by PixeKnight** [reply]
If you wanna know about any other xkb options, just type `man keyboard-config` in the terminal
